README.md

```diff
@@ -1,2 +1,15 @@
-# hw01
-Repository for homework 01
+# Homework 1  
+
+## Rachel Isaman 
+
+I am a Master's candidate in the *Center for Middle Eastern Studies* and a graduate research supervisor at the *Chicago Project on Security and Threats*. I'm originally from California and graduated in 2015 from UC Berkeley with a Bachelor's in History. My main research interest is security in conflict zones in the Middle East, particularly as it relates to cultural heritage protection. 
+
+## Places I Have Traveled
+1. Italy
+2. Spain
+3. France
+4. England 
+5. Scotland
+
+## Link to my Linked In 
+(https://www.linkedin.com/in/rachel-isaman-004b84132/)
```

Comment:
This creates a link, but a cleaner appearance would be something like:

```
[Link to my LinkedIn account](https://www.linkedin.com/in/rachel-isaman-004b84132/)
```

[Link to my LinkedIn account](https://www.linkedin.com/in/rachel-isaman-004b84132/)